ext/spl/spl_fixedarray.c

```diff
@@ -358,6 +358,20 @@ static zval *spl_fixedarray_object_read_dimension(zval *object, zval *offset, in
 
 	intern = Z_SPLFIXEDARRAY_P(object);
 
+	if (type == BP_VAR_IS) {
```

Comment:
Good point. IMHO we should follow current logic, simply call `offsetGet`, if `offsetGet` is overridden while `offsetHas` not. 
The reason why [bug #74478](https://bugs.php.net/bug.php?id=74478) is a bug is `$data['foo'] ?? 42;` equals to `isset($data['foo']) ? isset($data['foo']) : 42;`. We override `offsetHas` because we hope we can define how `isset()` works, and this bug ruled out the ability. If we don't override `offsetHas`, it indicates I'm pretty satisfied with how `isset()` works now, so we just need to follow current logic.